src/Ombi/Startup.cs

```diff
@@ -217,7 +217,14 @@ public IServiceProvider ConfigureServices(IServiceCollection services)
             app.UseSwagger();
             app.UseSwaggerUI(c =>
             {
-                c.SwaggerEndpoint("/swagger/v1/swagger.json", "My API V1");
+                if (settings.BaseUrl.HasValue())
+                {
+                    c.SwaggerEndpoint(settings.BaseUrl + "/swagger/v1/swagger.json", "My API V1");
```

Comment:
Excellent job! Just one thing!

Can we use string interpolation instead of string concatenation? 

For example $"{settings.BaseUrl}/swagger..."

Reply:
On it. I didn't think that C# had syntax like Python fstrings or js string literals.